### DISCUSSION TITLE
Great Program + Question

### DISCUSSION THREAD

**Discussion by ghost** [digitalw0lf/hextor#43] [General]
Congrats for the simply amazing program, it loads in the speed of light and uses 10 times less RAM than some other one I've tested here on github (while looking and feeling much better too).

A question: does this work on windows 7 SP1 or older?

**Comment by digitalw0lf** [comment]
I haven't tested it on Win7. It does NOT work on XP, but works on Win10 and under Wine on Ubuntu.
If you have some issues, please send me detailed description with message texts/screenshots.
